transcom/mymove
# B 18633 actual move date not future frontend integration

## B-18633

## Summary

As a customer uploading documents, future dates should be greyed out when setting the departure date.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Create a PPM move
2. As SC edit and submit move
3. As customer click Upload documents and select calendar under "Departure date", and verify all future dates are greyed out

## Screenshots
<img width="999" alt="Screenshot 2024-03-19 at 12 59 25 PM" src="https://github.com/transcom/mymove/assets/146969726/59265c37-dc9c-48cc-9d96-743534d8590f">
<img width="942" alt="Screenshot 2024-03-19 at 1 01 45 PM" src="https://github.com/transcom/mymove/assets/146969726/cfc0330d-6609-41aa-9b11-903f242e375c">
